spec/unit/userActivity.spec.js

```diff
@@ -0,0 +1,202 @@
+const { UserActivityTracker, UserActivityTrackerConfig  } = require("../../lib/components/userActivity")
```

Comment:
Had we managed to land https://github.com/matrix-org/matrix-appservice-bridge/pull/349, we could have made this typescripty. Ah well.

Reply:
/me puts in on The List ;)